app/assets/javascripts/govuk_publishing_components/vendor/lux/lux-polyfill.js

```diff
@@ -0,0 +1,152 @@
+/* eslint-disable */
```

Comment:
Is it worth putting a comment at the top of this file (and the other Lux file) to explain where it's from and how to get the latest version? I can imagine having a problem at some point in the future and someone trying to fix this, not knowing that it's not something we've written (I guess it is in `vendor` though).

Reply:
This script shouldn't change too often if at all - it's the equivalent of the Google Analytic's snippet that goes in the pages you want to track. 

The `lux.js` might change regularly - and has a version variable near the [top of the script](https://github.com/alphagov/govuk_publishing_components/blob/51fd6a286b2745f70cb061d9821aa7294e9206ac/app/assets/javascripts/govuk_publishing_components/vendor/lux/lux.js#L17).

I'll add a comment at the top of both scripts explaining what they're for, and pointing at the docs.